modules/ROOT/pages/getting-started.adoc

```diff
@@ -2,127 +2,96 @@
 
 == Prerequisites
 
-- Currently GitHub is the only supported git provider. You must have access to a GitHub repository that contains your policies and (optional) tests. This repository can be a private one. An example repository is available at https://github.com/cerbos/example-cerbos-policy-repository that you can use as a template if you are starting from scratch.
-- Cerbos version 0.31.0 or higher. Audit log collection requires Cerbos 0.36.0 or higher.
-- Outbound internet connection for your Cerbos instances so that they can connect to Cerbos Hub to fetch bundle updates and upload audit logs (optional).
+* A GitHub repository that contains your Cerbos policies and, optionally, tests. It can be private. If you are starting from scratch, use https://github.com/cerbos/example-cerbos-policy-repository as a template.
```

Comment:
I think this should be split up into two getting-started sections. One for those who have a GitHub repo and another for those who just have some policies that they could upload to Hub.